### PR TITLE
##To_Do_List with webpack

### PR DESCRIPTION
**Pull request by sarazaToussaint**
### In this project we: 

- Set up HTML-CSS-javascript linters
- Set up Webpack configurations
- Create a dynamic to-do list using objects

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+      <div id="${task.index}" class="task">
+        <div>
+            <input id="${task.index}" class="checkbox" type="checkbox" name="checkbox" ${!task.completed ? '' : 'checked'} />
+            <input id="task" type='text' class=" ${!task.completed ? '' : 'checked'} " value="${task.description}" />
+        </div>
+        <i id="ellips-btn" class="fa-solid fa-ellipsis-vertical ellips hidden"></i>
+        <i id="trash" class="fa-solid fa-trash trash"></i>
+    </div>
+      `).join('');
+};
```
Comment (TedLivist) [inline src/index.js:28]: - [ ] According to the project guidelines, you should have an array of objects with each object having the following properties: description `(string)`, completed `(boolean)`, index `(number)`. Kindly create this.

- [ ] Your function should dynamically render the created array on the page.

  tours

File: README.md [hunk 2]
```diff
@@ -1,73 +1,46 @@
...
-### Prerequisites
-
-### Setup
+- To get a local copy open and running follow these simple example steps.
 
-### Install
+   ## Setup
 
-### Usage
+   - Download or Clone Repository.
```
Comment (TedLivist) [inline README.md:20]: - [ ] Kindly add detailed instructions on how to clone the repo and `cd` into the directory.

File: README.md [hunk 3]
```diff
@@ -1,73 +1,46 @@
...
 
-### Run tests
+   ## Installation
```
Comment (TedLivist) [inline README.md:22]: - [ ] After running `npm install`, kindly include the specific command for running the app i.e. `npm run start`

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @sarazaToussaint,

Good job so far!
✔️ Nice UI design
✔️ Project is set up with webpack
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 3 above)
Comment (sarazaToussaint) [reply README.md:22]: Hi, @TedLivist I think you are doing well.
I made all the changes requested. Thank for your review

File: README.md [hunk 4]
```diff
@@ -1,73 +1,47 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
```
Comment (rustam-tolipov) [inline README.md:1]: - [x] [OPTIONAL] Good job 💥. Please also don't forget to add app screenshot to your readme

**Review by rustam-tolipov — APPROVED** [review]
## Hi @sarazaToussaint 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

image

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: README.md (hunk 4 above)
Comment (sarazaToussaint) [reply README.md:1]: Hi @Rustamxon7, thank you for approving my PR and mentioning this optional request.

**Pull request merged**